app/components/events/view/export/api-response.js

```diff
@@ -1,14 +1,58 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { htmlSafe } from '@ember/string';
-
+import ENV from 'open-event-frontend/config/environment';
 
 export default Component.extend({
+  modelID: computed('params.[]', function() {
+    return this.get('params')[0];
```

Comment:
@mayank8318 what's the console log output of this ?

Reply:
@CosmicCoder96 I get cannot read property 0 of undefined. I am now passing it as a name argument and the tests pass locally for me